hack/lib/common.sh

```diff
@@ -107,6 +107,12 @@ else
   sudo systemctl stop apparmor
 fi
 
+DEFAULT_NETWORK_TEMPLATE=${KUBE_ROOT}/hack/runtime/default_flat_network.json
+ARKTOS_NETWORK_TEMPLATE=${ARKTOS_NETWORK_TEMPLATE:-${DEFAULT_NETWORK_TEMPLATE}}
+if [ "${ARKTOS_NETWORK_TEMPLATE}" == "empty" ]; then 
```

Comment:
IMHO, the default behavior - at this moment - should be __not__ create default networks on new tenants, to keep backward compatability.